guide/07-working-with-bigdata/02-making-your-data-accessible-to-ga-server.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>If your data lies locally on your system, you can directly register it by passing the shared path of that data into the <code>add_bigdata</code> function. It is recommended that you use shared paths to allow other servers to access the data while running operations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Wording change in the last sentence,</p>